addon/components/suspense/index.js

```diff
@@ -78,7 +78,10 @@ export default class Suspense extends Component {
       return task;
     }
 
+    // TODO: Fix these linting errors, getters shouldn't have side effects
```

Comment:
The fix for this wasn't obvious, so I disabled it to keep linting passing. You'll want to come back and take a look at this later.